samples/contrib/azure-samples/databricks-pipelines/databricks_notebook_pipeline.py

```diff
@@ -0,0 +1,164 @@
+"""Import a notebook into a Databricks workspace and submit a job run to execute it in a cluster.
+Notebook will accept some parameters and access a file in DBFS and some secrets in a secret scope.
+"""
+from pathlib import Path
+import base64
+import kfp.dsl as dsl
+import kfp.compiler as compiler
+import databricks
+
+def create_dbfsblock(block_name):
+    return databricks.CreateDbfsBlockOp(
+        name="createdbfsblock",
+        block_name=block_name,
+        data="QWxlamFuZHJvIENhbXBvcyBNYWdlbmNpbw==",
+        path="/data/foo.txt"
+    )
+
+def create_secretscope(scope_name):
+    return databricks.CreateSecretScopeOp(
+        name="createsecretscope",
+        scope_name=scope_name,
+        initial_manage_principal="users",
+        secrets=[
+            {
+                "key": "string-secret",
+                "string_value": "helloworld"
+            },
+            {
+                "key": "byte-secret",
+                "byte_value": "aGVsbG93b3JsZA=="
+            },
+            {
+                "key": "ref-secret",
+                "value_from": {
+                    "secret_key_ref": {
+                        "name": "mysecret",
+                        "key": "username"
+                    }
+                }
+            }
+        ]
+    )
+
+def import_workspace_item(item_name):
+    current_path = Path(__file__).parent
+    notebook_file_name = current_path.joinpath("notebooks", "ScalaExampleNotebook")
+    notebook = open(notebook_file_name).read().encode("utf-8")
+    notebook_base64 = base64.b64encode(notebook)
+    return databricks.ImportWorkspaceItemOp(
+        name="importworkspaceitem",
+        item_name=item_name,
+        content=notebook_base64,
+        path="/Users/alejacma@microsoft.com/ScalaExampleNotebook",
```

Comment:
Do you want to wipe out your personal email here and add some comment to remind users to replace it?

Reply:
Done